composer.json

```diff
@@ -29,7 +29,7 @@
         "ext-hash": "*",
         "ext-intl": "*",
         "ext-json": "*",
-        "api-platform/core": "^2.6",
+        "api-platform/core": "^2.7.10",
```

Comment:
This version should be also bumped in ApiBundle, and conflicts with `2.7.0` could be removed now

Reply:
Done, thanks 🙏🏼